.circleci/config.yml

```diff
@@ -93,6 +101,16 @@ commands:
           working_directory: wordpress-develop/src/wp-content/plugins/atomic-blocks/
           command: composer install --prefer-dist --no-suggest --optimize-autoloader && npm install
 
+  update-node:
+    description: "Update the Node version"
+    steps:
+      - run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+          export NVM_DIR="/opt/circleci/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # Loads nvm.
+          [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion" # Loads nvm bash_completion.
+          nvm install --lts
```

Comment:
This seems pretty verbose to just bump the Node version, but I couldn't find a better way to do this, like with a parameter of `machine` or `image`